tests/api-mongo.test.js

```diff
@@ -466,6 +466,39 @@ describe('MongoDB Words', () => {
         });
     });
 
+    it('should return a list of igbo terms when using english by using single quotes', (done) => {
+      const keyword = '\'water\'';
+      getWords({ keyword })
+        .end((_, res) => {
+          expect(res.status).to.be.equal(200);
```

Comment:
This is the error I'm getting when I console `res.body`:

<img width="684" alt="Screen Shot 2021-04-26 at 10 14 23 AM" src="https://user-images.githubusercontent.com/16169291/116123698-2dd2ad00-a678-11eb-8359-60595221df9a.png">

Turns out that `replaceAll` doesn't work in the current working version of Node 12 and 14 (the function might be available in Node 15)

To get around this, use `replace` instead of `replaceAll` and use a RegExp:

```
req.query.keyword.replace(/["']/g, '');
```